doc/nrf/libraries/bluetooth_services/rpc.rst

```diff
@@ -83,53 +83,49 @@ Set the following options in the same way for the :ref:`ble_rpc_host` or :ref:`i
   * :kconfig:option:`CONFIG_BT_DEVICE_NAME`
   * :kconfig:option:`CONFIG_NRF_RPC_CBKPROXY_OUT_SLOTS` on one core must be equal to :kconfig:option:`CONFIG_NRF_RPC_CBKPROXY_IN_SLOTS` on the other.
 
-To keep all the above configuration options in sync, create an overlay file that is shared between the application and network core.
+To keep all the above configuration options in sync, create a snipptet with an overlay file that is shared between the application and network core.
```

Comment:
```suggestion
To keep all the above configuration options in sync, you can create a snippet with an overlay file that is shared between the application and network core.
```
Also consider linking to sysbuild snippet documentation: https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/build/snippets/index.html.